geode-core/src/main/java/org/apache/geode/management/internal/cli/domain/RegionDescriptionPerMember.java

```diff
@@ -63,7 +63,7 @@ public boolean equals(Object obj) {
           && this.getDataPolicy().equals(regionDesc.getDataPolicy())
           && this.isAccessor == regionDesc.isAccessor;
     }
-    return true;
+    return false;
```

Comment:
I agree that `false` is the more sensible choice for type mismatch, but I don't know that the `equals` for this class is ever actually used.  Should we just get rid of this method and let the class inherit `Object`'s?

Same for `hashCode`.

Reply:
Also, it's not in your diff so I can't comment on the actual line, but I'd love to see the empty javadoc in lines 28-30 be pruned.

Reply:
Done.